src/goto-symex/symex_function_call.cpp

```diff
@@ -288,7 +288,19 @@ void goto_symext::symex_function_call_post_clean(
 
   if(!goto_function.body_available())
```

Comment:
how does GOTO distinguish functions without bodies from functions with an empty body ?

Reply:
The implementation [checks whether the list of instructions is empty](https://github.com/diffblue/cbmc/blob/develop/src/goto-programs/goto_function.h#L35). Even in a function that has an "empty body", there will be an END_FUNCTION instruction at the very least.

Reply:
There a functions with empty bodies, like so `foo() { }`, and functions that aren't defined at all. The "empty body" ones do have the `END_FUNCTION`, whereas the undefined ones don't. An empty body is perfectly ok, and won't be affected by this change.